src/components/AppContent/AppContent.vue

```diff
@@ -24,14 +24,37 @@
 
 <template>
 	<main id="app-content-vue" class="app-content no-snapper">
-		<!-- @slot Provide content to the app content -->
-		<slot />
+		<div id="app-content-wrapper" :style="`height: ${contentHeight}px;`">
+			<!-- @slot Provide content to the app content -->
+			<slot />
+			<Splitpanes
+				class="default-theme"
+				@resized="handlePaneResize">
+				<Pane class="sidebar"
+					:size="paneSizes.sidebar || paneDefaults.sidebar.size"
+					:min-size="paneDefaults.sidebar.min"
+					:max-size="paneDefaults.sidebar.max">
+					<slot name="sidebar" />
+				</Pane>
```

Comment:
Sidebar is what we have on the far right of the apps, please use something like list or something?

Also, what happens if you don't have a list? Do we really need the Pane component to be used in that case?
Is it ignore? Maybe it would be better to have a v-if and v-else if there are no list slot to just output the main content?

Reply:
Hmmm. Well "list" is pretty specific to the usecase. It could be used for a preview of something for all we know. 

I like the idea with the v-if. What if we did that and verbosely call it `resizable-column`? 

So in case of a single column usage, we'd just have the default slot and if they want to add a resizable column, they just add that one named slot to it:

```
# Single main column
<AppContent>
   <div>Main content</div>
</AppContent>


# Two resizable columns
<AppContent>
   <div>Main content</div>
   <template slot="resizable-column">List of things</template>
</AppContent>
```

💡 ⚡ 

What do you think?

Reply:
Not really, we only have two allowed use-cases here: See my comment in https://github.com/nextcloud/mail/pull/4123#issuecomment-795618458

Either 
```vue
<AppContent>
	<AppContentList #list><!-- Mails listing --></AppContentList>
	<AppContentDetails #details><!-- Mail details --></AppContentDetails>
</AppContent>
```
Or
```vue
<AppContent>
	<!-- Main content -->
</AppContent>
```

Reply:
Also see https://docs.nextcloud.com/server/21/developer_manual/design/content.html

Reply:
Alright, I see. I changed it to list and details. ;)